packages/usa-pagination/src/styles/_usa-pagination.scss

```diff
@@ -119,7 +119,7 @@ $pagination-hover-token: list.nth($pagination-link-tokens, 2);
 
 .usa-pagination__button {
   align-items: center;
-  border-color: color.adjust(color($pagination-current-color), $alpha: -0.8);
+  border-color: color.adjust(color($pagination-current-color), $alpha: 0.6);
```

Comment:
**suggestion (blocking):** Looking into [sass:color adjust](https://sass-lang.com/documentation/modules/color/), this sass function increases or decreases the property of the color by fixed amounts.

By setting a positive value here instead of a negative value, we're increasing the alpha (opacity) by 60%. Since the color that's being passed in is already fully opaque, passing in a positive value does not affect the opacity. The result is a fully opaque border using the `$pagination-current-color`

Instead, I would recommend searching for a _negative_ value that will maintain the original behavior of implementing a slightly transparent border color that will pass the AA non-text contrast minimum value of `3:1`

> [!TIP]
> Once you find a value that works for the default background color, try looking into how `$pagination-current-color` is set and how you might be able to test this feature with different settings!

Reply:
I wonder if we should move away from opacity shifts here since we have determined that contrast is important here for accessibility. Should we move away from alpha transformations in `color.adjust` and instead use an explicit token here (maybe through [next-token() ](https://github.com/uswds/uswds/blob/develop/packages/uswds-core/src/styles/functions/color/next-token.scss) or some other method). I'll ask the question in Teams too in case we want to discuss this more there.

Reply:
Agreed — If LOE is minor we should look into using a theme token.

If a theme token is high LOE, I'm good with this solution. Moving to new functionality, like sass color adjust, could increase scope. We should also start relying less on SASS functions if we can avoid it.